core/jazz-ui/src/app/secondary-components/create-service/oss/create-service.component.html

```diff
@@ -397,6 +403,18 @@ <h4 class="bold-title">
                                 <p class="learn-more">Function would be triggered when an object is added or modified inside the bucket</p>
                             </div>
 
+                            <div class="sqs-fields row zero-margin arrow_box" *ngIf="chkSQS()">
+
+                              <div class="col-lg-12">
+                                  <div class="text">Stream ARN</div>
+                                  <div class="label-input slct-bx-selected input-wrapper">
+                                      <input type="text" disabled="disabled" class="inline-input input-label stream-sqs aws-events-left-block" value="arn:aws:sqs:us-west-2:302890901340:stream/">
```

Comment:
Can we use a generic number here (Account number in ARN)?

Reply:
Do we move this value to config file? 
we are using the same format for kinesis too

Reply:
Config file already has the account id, so perhaps you can use that.